workflows/main.wdl

```diff
@@ -111,7 +121,8 @@ workflow pmdbs_sc_rnaseq_analysis {
 					doublet_score_max = doublet_score_max,
 					total_counts_limits = total_counts_limits,
 					n_genes_by_counts_limits = n_genes_by_counts_limits,
-					allen_mtg_precomputed_stats = allen_mtg_precomputed_stats,
+					allen_brain_mmc_precomputed_stats_h5 = allen_brain_mmc_precomputed_stats_h5,
+					allen_brain_mmc_marker_genes_json = select_first([allen_brain_mmc_marker_genes_json]),
```

Comment:
This looks like it's optional throughout - you should just pass it directly, no select_first needed - in fact, this will fail at runtime if this file is _not_ provided (ie running the human path).